src/containers/List/ListWrapper.tsx

```diff
@@ -8,18 +9,20 @@ import { useLanguageProvider } from '../../contexts/Localisation'
 import { findUserRole, checkExistingUserRole } from '../../utils/helpers/list/findUserRole'
 import { useUserState } from '../../contexts/UserState'
 import { useMapColumnsByRole } from '../../utils/helpers/list/mapColumnsByRole'
-import { ApplicationListRow, ColumnDetails, SortQuery } from '../../utils/types'
+import { ApplicationListRow, ColumnDetails, FilterDefinitions, SortQuery } from '../../utils/types'
 import { USER_ROLES } from '../../utils/data'
 import { Link } from 'react-router-dom'
 import ApplicationsList from '../../components/List/ApplicationsList'
 import PaginationBar from '../../components/List/Pagination'
 import ListFilters from './ListFilters/ListFilters'
-import { useApplicationFilters } from '../../utils/data/applicationFilters'
+import { useGetFilterDefinitions } from '../../utils/helpers/list/useGetFilterDefinitions'
 import Loading from '../../components/Loading'
+import { usePrefs } from '../../contexts/SystemPrefs'
 
 const ListWrapper: React.FC = () => {
   const { strings } = useLanguageProvider()
-  const APPLICATION_FILTERS = useApplicationFilters()
+  const { preferences } = usePrefs()
+  const FILTER_DEFINITIONS = useGetFilterDefinitions(preferences?.defaultListFilters || [])
```

Comment:
Better I think to have ` const { preferences } = usePrefs()` in `useGetFilterDefinitions` rather than passing it in here. Hide all that stuff in the hook rather than cluttering this component, I'd say.

Reply:
Hmmm Ok - I'll have a go

Reply:
Nah, not simple to do. I'll keep as is

Reply:
You must have misunderstood what I was suggesting -- it's literally 2 minutes work and is much tidier. Please see comparison: https://github.com/openmsupply/conforma-web-app/compare/%231274-show-default-display-filters-on-top...%231274-show-default-display-filters-on-top-change-hook-input

Reply:
Very good. Thanks!